tests/entry_page_test.js

```diff
@@ -27,88 +27,18 @@ Scenario('Verify confirmations when both experiments matching', async ({ I }) =>
     })
 
     let experimentsConfig = await utils.getConfiguration();
- 
-    const entryPoints = await utils.getEntryPoints(activeKeys, experimentsConfig);
-   
-    //Assert
-    entryPoints.forEach(element => {
-        if(element.isEntryPoint){
-            I.assertContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }else{
-            I.assertNotContain(confirmations[entryPoints.indexOf(element)], element.id);
-        }
-    });
-});
-
-Scenario('Verify no confirmations when first experiment (non entry page) matching', async ({ I }) => {
```

Comment:
Looks like we're deleting 2 scenarios here -- can we bring them back?

Reply:
I explained in the PR description why I removed those tests.